docs/src/pages/docs/react-query/useSuspenseInfiniteQuery.en.mdx

```diff
@@ -20,7 +20,7 @@ const Example = () => {
 
 ### Motivation
 
-If you turn suspense on in @tanstack/react-query, You can use useInfiniteQuery with `<Suspense/>` and `<ErrorBoundary/>`.
+If you turn suspense on in @tanstack/react-query, You can use (useInfiniteQuery)[https://tanstack.com/query/v4/docs/react/reference/useInfiniteQuery] with `<Suspense/>` and `<ErrorBoundary/>`.
```

Comment:
This is right

```suggestion
If you turn suspense on in @tanstack/react-query, You can use [useInfiniteQuery](https://tanstack.com/query/v4/docs/react/reference/useInfiniteQuery) with `<Suspense/>` and `<ErrorBoundary/>`.
```

Reply:
Sorry @SEOKKAMONI It is my fault 😂

Reply:
No, it's because I didn't check it. sorry